test/README.md

```diff
@@ -1,16 +1,20 @@
 Tests
 =====
 
-Tests are run by `/cmake/RunTests.cmake` file, using `busted`.
+Tests are run by [`cmake/RunTests.cmake`](../cmake/RunTests.cmake).
```

Comment:
This was intentionally not linkified so that `gf` works.  The idea of putting these docs in the tree is so that people can view them locally. Web browser is the secondary case.

Reply:
How does linking it prevent `gf`?  This is actually more likely to work since it isn't an absolute path.